src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -121,7 +122,7 @@ private static ArgumentParser getArgumentParser() {
                 .nargs("*")
                 .metavar("FORMAT")
                 .type(new AlphanumericArgumentType())
-                .setDefault(Format.DEFAULT_FORMAT_STRINGS)
+                .setDefault(new ArrayList<String>())
```

Comment:
If we are not adding elements into the `ArrayList`, use `Collections.emptyList()`